src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -322,11 +348,17 @@ def get_graph(self, secrets: Dict[str, Any]) -> Dataset:
         collections = []
         for endpoint in self.endpoints:
             fields: List[Field] = []
-            read_request = endpoint.requests.get("read")
-            delete_request = endpoint.requests.get("delete")
```

Comment:
There's a bunch of these, changing from `requests.get("delete")` to `requests.delete`